woocommerce/woocommerce-ios
# Gridicons: All gridicons should be stored in the `UIImage+Woo` extension

Fixes #746 

Encapsulate all access to GridIcons in the UIImage+Woo extension. Considering the announcements at WWDC, this seemed like the right moment to do this 😄 

## Changes
- Added vars to the images extension, and reordered the existing ones in alphabetical order (sorry about that 😊 )
- Added tests to ensure the integrity of the dependency

## Testing
- Checkout the branch, run the tests. The tests should pass
- Search the codebase for `Gridicon.iconOfType`. All occurrences should be only in `UIImage+Woo` (and a few results in Pods, out of the scope of this story)
- Navigate around, check that icons are still rendered and that I haven't messed up anything
